### ISSUE TITLE
Atrophic vaginal mucosa

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10107]
**Preferred term label:**
Atrophic vaginal mucosa

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C2230359

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Menopause

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000067
- label: Atrophic vaginal mucosa
- parents: [Abnormal vagina morphology (HP:0000142)](https://hpo.jax.org/app/browse/term/HP:0000142)
- pmid: PMID:33232011
- def: Atrophic vaginal changes are caused by thinning vaginal epithelium, decreased vaginal rugae and elasticity, and decreased vaginal secretions.
- comment: This finding is typically observed in postmenopausal women, with symptoms appearing in the early to the late postmenopausal stage. Women with other causes of low estrogen levels may also be affected.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10107
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
HPO ID corrected to HP:0040413
